molparse/system.py

```diff
@@ -69,6 +70,32 @@ def autoname_chains(self, verbosity=1):
                     f"Ambiguous naming! Multiple chains named {chain.name}!"
                 )
 
+    def ssbond_guesser(self, distance=2.15):
+        """Generate SS bond between SG of CYS according to distances"""
+        import numpy as np
+
+        cysteines=[res for res in self.residues if res.name == "CYS"]
+        start=0
+        end=len(cysteines)
+        for i in range(start, end-1):
+            # Look for x, y, z coordinates of SG for cysteine 1
+            for atom in cysteines[i].atoms:
+                if atom.name == "SG":
+                    sg1_pos = np.asarray(atom.position)
+            for j in range(start+1,end):
+                # Look for x, y, z coordinates of SG for cysteine 2
+                for atom in cysteines[j].atoms:
+                    if atom.name == "SG":
+                        sg2_pos = np.asarray(atom.position)
+                # Calculate distance between both SGs
+                dist = np.linalg.norm(sg1_pos - sg2_pos)
```

Comment:
`Molparse.Atom` objects support subtraction to calculate their distance:

```
sg1 = cysteine1.get_atom("SG")
sg2 = cysteine2.get_atom("SG")
dist = sg1 - sg2
```

See `Atom.__sub__`